app/javascript/stylesheets/application.scss

```diff
@@ -1,9 +1,12 @@
 $govuk-assets-path: "~govuk-frontend/govuk/assets/";
 $govuk-font-family: "GDS Transport" , arial, sans-serif !important;
+$bg-orange: #FFA300 !important;
```

Comment:
Hey just fyi we have a `colours.scss` file so might be better to have this in there 😃 . And do we definitely need the `!important` flag here ?